flow-sample/src/main/java/com/example/flow/DemoApp.java

```diff
@@ -38,10 +28,6 @@
     globalGraph = ObjectGraph.create(new DaggerConfig());
```

Comment:
If we get dagger out of here we can drop the custom app entirely! Not in this PR, obviously.